lib/chef/knife/core/node_presenter.rb

```diff
@@ -125,7 +125,7 @@ def summarize(data)
             summarized << <<-SUMMARY
 #{key('Recipes:')}     #{Array(node[:recipes]).join(', ')}
 #{key('Platform:')}    #{node[:platform]} #{node[:platform_version]}
-#{key('Tags:')}        #{Array(node[:tags]).join(', ')}
+#{key('Tags:')}        #{node.tags.join(', ')}
```

Comment:
We're super sure now that node.tags will always give us an array, because that's part of the bug, right?